File: lock cracker.java
class Main {

    public static void lock() {
        
        String lock = "750";

        for (int i = 0; i < 10; i++) {
            for (int j = 0; j < 10; j++) {
                for (int k = 0; k < 10; k++) {

                    String i2 = String.valueOf(i);
                    String j2 = String.valueOf(j);
                    String k2 = String.valueOf(k);

                    String output = i2 + j2 + k2;

                    System.out.println(output);

                    if (output.equals(lock)) {
                        System.out.println("lock combination found: " + output);
                        return;
                    }
                        
                }
            }
        }

        System.out.println("combination not found");
    }
    
    public static void main(String[] args) {
        lock();
    }
}

File: lock cracker.py
def lock():

    lock = "225"
    
    for i in range(0, 10):
        for j in range(0, 10):
            for k in range(0, 10):

                i2 = str(i)
                j2 = str(j)
                k2 = str(k)

                output = i2 + j2 + k2

                print(output)

                if output == lock:
                    print(f"lock combination found: {output}")
                    return

    print("combination not found")

def main():
    lock()

if __name__ == '__main__':
    main()
